vnext/Microsoft.ReactNative/ReactPointerEventArgs.idl

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "DocString.h"
+#include "NamespaceRedirect.h"
+
+namespace Microsoft.ReactNative {
+  [webhosthidden]
+  DOC_STRING("Event arguments wrapper for IViewManagerWithPointerEvents.")
+  runtimeclass ReactPointerEventArgs {
+    DOC_STRING("Prevent event from being emitted to React JS component.")
+    XAML_NAMESPACE.Input.PointerRoutedEventArgs Args {
```

Comment:
this still ties us to a xaml dialect, might be ok for now but if we ever want to be truly agnostic we'll need to expose the properties we want outside of exposing a xaml object

Reply:
Indeed, we'd probably need the rough equivalent of whatever we forward in the RN pointer events, so things like isLeftButton / isRightButton / keyModifiers / pagePoint / screenPoint / offsetPoint / etc., but we can't determine some of these (namely, "offsetPoint") until we know the React target. I also don't want to design for an abstraction we don't have yet.